src/Language/PureScript/Errors.hs

```diff
@@ -781,6 +771,14 @@ prettyPrintSingleError full level e = do
       paras [ line $ "The import of type " ++ runModuleName name ++ " as " ++ runModuleName qualName ++ " uses the deprecated 'import qualified' syntax."
             , line $ "This syntax form will be removed in PureScript 0.9." ]
 
+    renderSimpleErrorMessage (DeprecatedClassImport mn name) =
+      paras [ line $ "The import of class " ++ runProperName name ++ " from " ++ runModuleName mn ++ " uses the deprecated syntax that omits the class keyword."
```

Comment:
I'll go ahead and make this change, it's certainly no worse than my original error message.